pdns/dnsdist-lua.cc

```diff
@@ -158,7 +158,7 @@ std::unordered_map<int, vector<boost::variant<string,double>>> getGenResponses(u
   vector<pair<int, DNSName>> rcounts;
   rcounts.reserve(counts.size());
   for(const auto& c : counts) 
-    rcounts.push_back(make_pair(c.second, c.first));
+    rcounts.push_back(make_pair(c.second, c.first.makeLowerCase()));
```

Comment:
I wonder if it wouldn't be better to lowercase a bit later then, replacing `rc.second.toString()` by something like `rc.second.makeLowerCase().toString()`, so we only do the lowercasing for the names we are going to print, not all of them.